CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 ### Added
+- Amended time aggregation parameter names used in existing `total_network_objects` query from "period" in flowclient & "by" in flowmachine to "total_by" in both.
```

Comment:
Sorry to be pedantic, but can this go in the changed section of [Unreleased], and can we explicitly refer to `TotalNetworkObjects` and `AggregateNetworkObjects`?